### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by nijinghan** [nijinghan/MI-349-FS17-740-css-frameworks-p1twXJ#1]
## Build a homepage for an imaginary company (you can make up the product/service they offer)

@jsiarto Can you take a look at this? It's [hosted here](https://nijinghan.github.io/MI-349-FS17-740-css-frameworks-p1twXJ/) and meets the following criteria:

- [x] Include the CSS and JS for Bootstrap 3 (not 4) in the page (including the "jQuery" dependency)
- [x] Use the `container` class to center your page content
- [x] Use a `navbar` class to display the name of your site and a list of at least 3 links (to past projects, or anything else) at the top of your page
- [x] Split your main content into 2 columns on large (`lg`) and medium (`md`) sized screens, but stack them on top of each other on small (`sm`) and extra small (`xs`) screens
- [x] Use a `thumbnail` class on an at least one image in your page

**Comment by jsiarto** [comment]
This needs to be hosted on GitHub pages. Please publish and resubmit.

**Comment by nijinghan** [comment]
NiJinghans-MacBook-Pro:MI-349-FS17-740-css-frameworks-p1twXJ nijinghan$ git push origin master:gh-pages
Everything up-to-date

This is what the terminal told me, should be fixed. I think I did the same thing before not sure what happened. Thank you, please let me know @jsiarto

**Comment by jsiarto** [comment]
Yes, it looks good -- just a note: make sure you name the default page `index.html` so it shows at the default URL on GitHub Pages. :shipit:
